lib/IRGen/ClassMetadataVisitor.h

```diff
@@ -65,8 +65,10 @@ template <class Impl> class ClassMetadataVisitor
     // FIXME: Figure out what can be removed altogether in non-objc-interop
     // mode and remove it. rdar://problem/18801263
```

Comment:
Does this not address this FIXME?

Reply:
I'm not sure and that's why I asked at the top: "what other fields are good candidates for removal from the non-ObjC runtime?" 

I think that @rjmccall and @jckarter got too excited about other things that need fixing, so they missed that question. ;-)

Reply:
The three extra padding fields seem like the biggest obvious thing to remove. I can't off the top of my head think of any other places where we could wholesale remove fields. I'd be fine removing the fixme.